tests/Unit/Evolution/Systems/Cce/Actions/Test_CalculateScriInputs.cpp

```diff
@@ -83,30 +79,29 @@ struct mock_characteristic_evolution {
       ::Tags::Variables<
           tmpl::append<all_boundary_pre_swsh_derivative_tags_for_scri,
                        all_boundary_swsh_derivative_tags_for_scri>>>;
-  using compute_tags = db::AddComputeTags<>;
-
-  using initialize_action_list =
-      tmpl::list<ActionTesting::InitializeDataBox<simple_tags, compute_tags>>;
-  using initialization_tags =
-      Parallel::get_initialization_tags<initialize_action_list>;
 
   using metavariables = Metavariables;
   using chare_type = ActionTesting::MockArrayChare;
   using array_index = size_t;
+
+  // so that the precomputation mutator can be used in the tmpl::transform
+  template <typename Tag>
+  using local_precompute = PrecomputeCceDependencies<Tags::BoundaryValue, Tag>;
   using phase_dependent_action_list = tmpl::list<
-      Parallel::PhaseActions<typename Metavariables::Phase,
-                             Metavariables::Phase::Initialization,
-                             initialize_action_list>,
+      Parallel::PhaseActions<
+          typename Metavariables::Phase, Metavariables::Phase::Initialization,
+          tmpl::list<ActionTesting::InitializeDataBox<simple_tags>>>,
       Parallel::PhaseActions<
           typename Metavariables::Phase, Metavariables::Phase::Evolve,
-          tmpl::flatten<tmpl::list<
-              // extra pre swsh derivative tags
-              ::Actions::MutateApply<
-                  PreSwshDerivatives<Tags::Dy<Tags::Dy<Tags::BondiJ>>>>,
-              // extra precomputation tags
-              ::Actions::MutateApply<PrecomputeCceDependencies<
-                  Tags::BoundaryValue, Tags::OneMinusY>>,
-              Actions::CalculateScriInputs>>>>;
+          tmpl::list<tmpl::transform<
+                         extra_pre_swsh_derivative_scri_tags,
+                         tmpl::bind<::Actions::MutateApply,
+                                    tmpl::bind<PreSwshDerivatives, tmpl::_1>>>,
+                     tmpl::transform<
+                         extra_precomputation_scri_tags,
+                         tmpl::bind<::Actions::MutateApply,
+                                    tmpl::bind<local_precompute, tmpl::_1>>>,
+                     Actions::CalculateScriInputs>>>;
```

Comment:
Do you remember if the test passed before this fixup commit? If so, can you add checks that could have caught this issue?

Reply:
it did; this alteration doesn't change any of the execution of this test. This just causes the two `MutateApply` actions to be determined from the two typelists above `extra_pre_swsh_derivative_scri_tags` and `extra_precomputation_scri_tags` instead of manually determined. This is so that if we add more things to the scri computation input typelists, this test can be altered only by the corresponding additions to those above typelists, rather than finding everywhere those tags would get used implicitly.